src/main/java/org/irods/Util/Operation.java

```diff
@@ -0,0 +1,18 @@
+package org.irods.Util;
+
+/**
+ * enum for modifyMetadata() method in collections, data objects, resources, and users groups endpoint
+ */
+public enum Operation {
```

Comment:
`Operation` is very generic.

Let's name this `MetadataOperation`.

Reply:
Done.